common/anomalies/hfe_anomaly.txt

```diff
@@ -9,12 +10,13 @@ hfe_anomaly_wormhole = {
 	max_once_global = yes
 	null_spawn_chance = @null_spawn_chance
 	spawn_chance = {
+		base = 0
 		modifier = {
-			add = 1
+			add = 5
```

Comment:
this anomaly have small base chance because triger do not restrict planet types
5 will make it appear very fast

Reply:
Okay, done